### PR TITLE
Remove test job from front end

### LINKED ISSUE #25: BUG Remove Test Job
## User Story
As a user, I will:
- not be able to see the test job during production builds.

## Screenshots
Screenshot 2023-08-11 at 1 48 10 AM

## Discussion
I'm unsure why the test job is still visible during the prod build. We should fix this so that if we still want it, it should only be visible during testing.

### PR DESCRIPTION
**Pull request by dscpsyl**
## Overview

In this PR, the test job function is removed from the user-facing front end. It is still available in the Swagger backend. 

## Screenshots (Optional)
Screenshot 2023-08-22 at 3 04 52 PM

## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go to this page.
4. Test this function.

## Tests

- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes #25

### CODE REVIEW

**Review by publicqi — COMMENTED** [review]
Should we just delete the code to avoid confusion for future developers or commenting out the code?

Other than that, the code looks good to me.

**Comment by chriscerie** [comment]
> Should we just delete the code to avoid confusion for future developers or commenting out the code?

Yes. There’s very few cases where commented code is beneficial to readers, and this is not one of them.

File: frontend/src/main/pages/AdminJobsPage.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
 import PagedJobsTable from "main/components/Jobs/PagedJobsTable";
 import Accordion from "react-bootstrap/Accordion";
-import TestJobForm from "main/components/Jobs/TestJobForm";
+// import TestJobForm from "main/components/Jobs/TestJobForm";
```
Comment (pconrad) [inline frontend/src/main/pages/AdminJobsPage.js:5]: Ack!  What have we said about commenting out code in production?

It's fine to comment out while testing, but when you make a PR, the commented out code should go bye-bye.

File: frontend/src/main/pages/AdminJobsPage.js [hunk 2]
```diff
@@ -16,21 +16,21 @@ const AdminJobsPage = () => {
 
   // *** Test job ***
 
-  const objectToAxiosParamsTestJob = (data) => ({
-    url: `/api/jobs/launch/testjob?fail=${data.fail}&sleepMs=${data.sleepMs}`,
-    method: "POST",
-  });
+  // const objectToAxiosParamsTestJob = (data) => ({
```
Comment (pconrad) [inline frontend/src/main/pages/AdminJobsPage.js:19]: here too

File: frontend/src/main/pages/AdminJobsPage.js [hunk 3]
```diff
@@ -138,10 +138,10 @@ const submitInstructorReportSpecificCommonsJob = async (data) => {
 }
 
 const jobLaunchers = [
-  {
-    name: "Test Job",
-    form: <TestJobForm submitAction={submitTestJob} />,
-  },
+  // {
```
Comment (pconrad) [inline frontend/src/main/pages/AdminJobsPage.js:141]: and here

**Review by pconrad — CHANGES_REQUESTED** [review]
shaking my head... we've talked about commented out code, have we not?

**Comment by dscpsyl** [comment]
That is my bad. This PR is not ready yet. I'll close this PR and recreate one. There's still more testing to be done and I'm still experimenting

**Pull request closed without merging**